probe/docker/container.go

```diff
@@ -327,7 +332,9 @@ func (c *container) GetNode(hostID string, localAddrs []net.IP) report.Node {
 	if c.container.State.Paused {
 		result = result.WithControls(UnpauseContainer)
 	} else if c.container.State.Running {
-		result = result.WithControls(RestartContainer, StopContainer, PauseContainer)
+		result = result.WithControls(
+			RestartContainer, StopContainer, PauseContainer, AttachContainer, ExecContainer,
```

Comment:
That correct - I think the UI is sorting them alphabetically, which happens to looks good right now,